docs/dynamic-plugins/versions.md

```diff
@@ -0,0 +1,132 @@
+# Version Compatibility Matrix
+
+## RHDH 1.4 (pre-release, versions can change for final release)
+
+<!-- source
+https://github.com/janus-idp/backstage-showcase/blob/main/backstage.json
+-->
+
+Based on [Backstage 1.31.3](https://backstage.io/docs/releases/v1.31.0)
+
+To bootstrap Backstage app that is compatible with RHDH 1.3, you can should use:
+
+```bash
+npx @backstage/create-app@0.5.17
```

Comment:
Is the create-app same version as for RHDH 1.3? (0.5.17)

Reply:
yes, at the time of writing this it was. But it will most likely change for final release